testsuite/Rakefile

```
@@ -129,4 +146,111 @@ namespace :utils do
     sh 'find . -size 0 -delete', verbose: false
     sh 'node index.js &> cucumber_reporter.log', verbose: false
   end
+
+end
+
+namespace :jenkins do
+
+  desc 'Generate rake file runstep'
+  task :generate_rake_run_step, [:extension, :client_name, :output_path, :yaml_path] do |t, args|
```

```suggestion
  task :generate_rake_run_step, [:extension, :client_name, :output_path, :yaml_path] do |_t, args|
```
I cannot see this variables used anywhere below.